src/plugins_types.c

```diff
@@ -1638,38 +1727,25 @@ ly_type_store_instanceid(const struct ly_ctx *ctx, const struct lysc_type *type,
     /* store JSON string value */
     str = (char *)ly_type_print_instanceid(storage, LY_PREF_JSON, NULL, &dyn);
     assert(str && dyn);
-    LY_CHECK_GOTO(ret = lydict_insert_zc(ctx, str, &storage->canonical), error);
-
+    LY_CHECK_GOTO(ret = lydict_insert_zc(ctx, str, &storage->canonical), cleanup);
     storage->realtype = type;
 
     /* cleanup */
+cleanup:
     lyxp_expr_free(ctx, exp);
     ly_path_free(ctx, path);
 
+clean_value:
```

Comment:
s/clean_value/cleanup_value

Reply:
Thanks. I have forgotten about that when I made all changes.